packages/newsroom-signup/src/SmartContract/AboutSmartContractsButton.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import { InvertedButton, buttonSizes } from "@joincivil/components";
+import { urlConstants } from "@joincivil/utils";
+import { InfoModalButton, Header, SectionHeader, Paragraph } from "../InfoModalButton";
+
+export class AboutSmartContractsButton extends React.Component {
+  public render(): JSX.Element | null {
+    return (
+      <InfoModalButton
+        buttonText="Learn more about Newsroom Smart Contracts"
+        content={
+          <>
+            <Header>About Newsroom Smart Contracts</Header>
+            <Paragraph>
+              In this section, we explain the difference between the Smart Contract Address and Public Address and why
+              this is important to your Newsroom.
+            </Paragraph>
+
+            <SectionHeader>Newsroom Smart Contract</SectionHeader>
+            <Paragraph>
+              Smart contracts help you record activity and exchange things of value in a transparent, conflict-free way
+              while avoiding intermediary services. The public can find a newsroom by its contract address in order to
+              see its activities on the blockchain. You cannot send funds directly to a newsroom smart contract.
+            </Paragraph>
+            <Paragraph>
+              In the case of Civil Registry and Publisher, this newsroom smart contract code allows you to create and
+              maintain a decentralized index and/or archive of your content and control who has permission to update it.
+              In combination with your newsroom wallet address, the smart contract acts as a verification for your
+              newsroom's identity.
+            </Paragraph>
+            <Paragraph>
+              Your index will appear in the Civil network, and provides a way to track it back to your website, along
```

Comment:
does this really belong here,  also seems like an abrupt transition to bring up index and archive without introducing the terms

Reply:
The previous paragraph gives context with "this newsroom smart contract code allows you to create and maintain a decentralized index and/or archive of your content" which makes sense (and is still pretty technically accurate) using the dictionary definitions of those terms. I would maybe update this sentence to say "Using Civil's publishing tools, your content can be indexed in the Civil network, which provides..." and then we're probably good without getting into the precise meanings of these terms in our context. @picocassette what do you think?

Reply:
Yeah, sounds good to me @tobek